board/app/controllers/user_controller.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+class UserController extends AppController{
+
+    public function register() 
+    {
+        if (is_logged_in()) 
+        {
```

Comment:
if-statements format should be like this po:
```
if (condition) {
    ...
}
```
Please revise during your Project Course :bow: